testFlight1/lib/teensyShared/thresholds.h

```diff
@@ -1,18 +1,25 @@
 #ifndef THRESHOLDS_H
 #define THRESHOLDS_H
-
+ 
 //TODO set values for the thresholds
-float coast_time_thresh = 500; // .5 seconds, probably maybe
-float coast_thresh = -9; //"-9" because below -9.81 must be freefall, add some buffer for data noise
+float coast_time_thresh = 300;
+float coast_thresh = 0;
 float apogee_thresh = 0;
+float main_thresh_min = -2; //still not determined
+float main_thresh_max = -1; //still not determined
+float drogue_thresh = -2; //still not determined
```

Comment:
Either we check this in OpenRocket now, or we add a comment here to check the final OpenRocket and update it then. Either way, we'll have to check some of these thresholds before flight.